src/crud.js

```diff
@@ -0,0 +1,61 @@
+const tasklistDisplay = document.querySelector('.shores');
+const inputValu = document.querySelector('.newTask');
+const addBtn = document.querySelector('.addtolist');
+
+let taskList = [];
+
+const display = () => {
+  tasklistDisplay.innerHTML = '';
+  taskList = JSON.parse(localStorage.getItem('localItem')) || [];
+  taskList.forEach((task) => {
+    const tasklistDisplay = document.querySelector('.shores');
+    const shore = document.createElement('div');
+    shore.classList.add('listDo');
+    shore.innerHTML = `
+  <div class="listElement">
+  <input type="checkbox">
+  <h2>${task.description}</h2>
+  </div>
+  <div>
+  <i class="fa-solid fa-trash-can" id="${task.index}"></i>
+  </div>`;
+    tasklistDisplay.appendChild(shore);
+  });
+};
+
+display();
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (inputValu.value === '') return;
+  taskList = JSON.parse(localStorage.getItem('localItem')) || [];
+  const obj = {
+    description: inputValu.value,
+    completed: false,
+    index: taskList.length,
+  };
+
+  taskList.push(obj);
+  localStorage.setItem('localItem', JSON.stringify(taskList));
+  inputValu.value = '';
+  display();
+});
+
+const removeTask = (index) => {
```

Comment:
- [ ] For this milestone you're implementing CRUD ( Create Read Update and Delete ) operations. So, Kindly make sure that a user can edit or update an item description. For that, you have to implement a function for editing tasks description.

Reply:
thanks @RolandM99  you right .